qemu/tests/cfg/cpu_topology_test.cfg

```diff
@@ -13,3 +13,15 @@
             required_qemu = [6.2.0,)
             # Check PPTT table mapping by looking at core_id
             check_core_id_cmd = 'cat /sys/devices/system/cpu/cpu%s/topology/core_id'
+        s390x:
+            no RHEL.6 RHEL.7
+            no Host_RHEL.m6 Host_RHEL.m7 Host_RHEL.m8
+            required_qemu = [8.2.0,)
+    variants:
+        - check_siblings_core_id_cmd:
+            s390x:
+                RHEL.8:
+                    check_siblings_cmd = 'awk -F'-' '{print $2 - $1 + 1}' /sys/devices/system/cpu/cpu0/topology/core_siblings_list'
+                RHEL.9:
+                    check_siblings_cmd = 'cat /proc/cpuinfo |grep siblings |uniq |cut -d ":" -f 2'
+                check_core_id_cmd = 'cat /sys/devices/system/cpu/cpu%s/topology/core_id'
```

Comment:
`check_core_id_cmd = 'cat /sys/devices/system/cpu/cpu%s/topology/core_id'`
This configuration is also in s390. Is it set for rhel7 guest?